test/IECore/BlindDataHolder.py

```diff
@@ -124,6 +127,9 @@ def testLoadSave(self):
 		g2 = Object.load( iface, "test" )
 		self.assertEqual( g1, g2 )
 		
+		# "blindData" entry should be excluded from the IndexedIO hierarchy
+		self.assertEqual( iface.directory( ["test","data","Group", "data", "blindData"], IndexedIO.MissingBehaviour.NullIfMissing ), None )
```

Comment:
I think this should just be testing for "test/data/BlindDataHolder" - if the blind data were to be written it'd be into the BlindDataHolder container rather than the Group one. And we should test for the higher level directory, because excluding that gives the biggest storage optimisation.